javascript/node/selenium-webdriver/bidi/browsingContext.js

```diff
@@ -165,6 +166,86 @@ class BrowsingContext {
     const response = await this.bidi.send(params)
     return new PrintResult(response.result.data)
   }
+
+  async captureScreenshot() {
+    let params = {
+      method: 'browsingContext.captureScreenshot',
+      params: {
+        context: this._id,
+      },
+    }
+
+    const response = await this.bidi.send(params)
+    this.checkErrorInScreenshot(response)
+    return response['result']['data']
+  }
+
+  async captureBoxScreenshot(x, y, width, height) {
+    if (width < 0) {
+      x = x + width
+      width = -width
+    }
+
+    if (height < 0) {
+      y = y + height
+      height = -height
+    }
+
+    let params = {
+      method: 'browsingContext.captureScreenshot',
+      params: {
+        context: this._id,
+        clip: {
+          type: 'viewport',
+          x: x,
+          y: y,
+          width: width,
+          height: height,
+        },
+      },
+    }
+
+    const response = await this.bidi.send(params)
+    this.checkErrorInScreenshot(response)
+    return response['result']['data']
+  }
+
+  async captureElementScreenshot(
+    shareId,
```

Comment:
I think this is a typo. The spec refers to it as "sharedId". https://w3c.github.io/webdriver-bidi/#type-script-SharedId